bindings/python/CMakeLists.txt

```diff
@@ -176,7 +182,7 @@ FIND_PACKAGE (pybind11 2.6.0 REQUIRED)
 ### Python
 
 # Python versions to build for and support
-SET (PYTHON_SEARCH_VERSIONS 3.8 3.9 3.10 3.11 CACHE STRING "Versions of Python bindings to build.")
+SET (PYTHON_SEARCH_VERSIONS 3.8 3.9 3.10 3.11 3.12 CACHE STRING "Versions of Python bindings to build.")
```

Comment:
When is it appropriate to drop support for 3.8?

Reply:
Maybe once Python 3.8 is End-of-Lifed later this year?